src/constants/deviceParameters/channelsConfigs.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Possible values of channels config
+ *
+ * @packageDocumentation
+ */
+
+export const CHANNELS_ENABLED_1_2_3_4_DISABLED_SERIAL = 0;
```

Comment:
do we really need these constants? they look terrible.

how about info table in jooby-docs and a link here to docs?

Reply:
constants - comfortable, names - representation of that complex format, and no idea how to keep it in constants without these names

Reply:
constants removed